test/notBoth.js

```diff
@@ -0,0 +1,47 @@
+import S from 'sanctuary';
+
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
```

Comment:
insert additional empty line here pls. Would be great to have this rule in eslint rules. Always 2 lines between imports and the actual code.

Reply:
Yep. Would be good to lint for this.